cmake/Modules/UseWarnings.cmake

```diff
@@ -11,6 +11,9 @@ if (CXX_COMPAT_GCC)
 	message (STATUS "All warnings enabled: ${_warn_flag}")
 	add_options (ALL_LANGUAGES ALL_BUILDS "${_warn_flag}")
   endif (_warn_flag)
+  if(CMAKE_CXX_COMPILER_ID MATCHES "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 13)
```

Comment:
Is there a reason for not using `VERSION_GREATER_EQUAL` here?

Reply:
uhm, other than it being longer to type, not really. since 13 evaluates to 13.0.0 and 13.1.0 is the first gcc release it doesn't matter in this case. but sure, i haved changed it.